agent/util-scripts/test-bin/test-start-stop-tool-meister

```diff
@@ -56,17 +56,21 @@ printf -- "Exit code: %d\n" ${?} >> ${_ptms_output}
 cat -n ${_ptms_output}
 rm ${_ptms_output}
 
+function _timeout {
+    timeout --kill-after=10 --signal TERM 20 $*
+}
+
 # Now execute the success case.
 source ${_script_path}/common-tm-cleanup
 
-_PBENCH_TOOL_MEISTER_LOG_LEVEL="debug" _PBENCH_TOOL_DATA_SINK_LOG_LEVEL="debug" _PBENCH_TOOL_MEISTER_START_LOG_LEVEL="debug" PBENCH_REDIS_SERVER="localhost" PBENCH_TOOL_DATA_SINK="localhost:8080;localhost:8080" pbench-tool-meister-start --sysinfo="none" ${group}
+_PBENCH_TOOL_MEISTER_LOG_LEVEL="debug" _PBENCH_TOOL_DATA_SINK_LOG_LEVEL="debug" _PBENCH_TOOL_MEISTER_START_LOG_LEVEL="debug" PBENCH_REDIS_SERVER="localhost" PBENCH_TOOL_DATA_SINK="localhost:8080;localhost:8080" _timeout pbench-tool-meister-start --sysinfo="none" ${group}
```

Comment:
This line would be much more comprehensible if it were wrapped:
```
_PBENCH_TOOL_MEISTER_LOG_LEVEL="debug" \
    _PBENCH_TOOL_DATA_SINK_LOG_LEVEL="debug" \
    _PBENCH_TOOL_MEISTER_START_LOG_LEVEL="debug" \
    PBENCH_REDIS_SERVER="localhost" \
    PBENCH_TOOL_DATA_SINK="localhost:8080;localhost:8080" \
    _timeout pbench-tool-meister-start --sysinfo="none" ${group}
```
Ditto, line 73.